src/index.html

```diff
@@ -22,6 +22,8 @@
 		<meta name="google-site-verification" content="BtZGDxnyQEGEnP6itX9hcI29afbpdGaQ38RPPqe5r60" />
 		<meta name="p:domain_verify" content="3eff3c0d54abed5af5f00fc8c9bb8e32" />
 
+		<link rel="stylesheet" href="css/theme.css">
```

Comment:
@ilPas dato che usi webpack per buildare "solo" il contenuto della cartella `./scripts/`, webpack stesso non tocca il file index.html quindi dobbiamo linkare noi il file CSS.

`webpack-dev-server` hosta in automatico il file CSS nel processo live - per quello funzionava solo in DEV

Reply:
Qua si dovrebbe aggiungere un check per DEV vs PROD build perché la dev build da errore (mancando il file reale nel webpack-dev-server)